[OSDOCS#17136]: Changes as per CQA guidelines (#111163)
openshift/openshift-docs: microshift_troubleshooting/microshift-cleanup-data.adoc

:_mod-docs-content-type: ASSEMBLY
[id="microshift-cleanup-data"]

> = Clean up data with support

Reviewer: leave procedural assemblies as gerunds; do not change

include::_attributes/attributes-microshift.adoc[]
:context: microshift-cleanup-data

toc::[]

[role="_abstract"]

> {microshift-short} provides the `microshift-cleanup-data` script for troubleshooting tasks such as deleting data, certificates, and container images.

Reviewer: ```suggestion
You can use the `microshift-cleanup-data` script for troubleshooting tasks such as deleting data, certificates, and container images.
```
user-focus
